src/podman.scss

```diff
@@ -1,3 +1,9 @@
+/* Collapse top padding with additional container fluids */
+/* FIXME: This should go into PF overrides in Cockpit. */
+/* Remove it here when it lands there. */
+.container-fluid + .container-fluid {
```

Comment:
I don't understand the + here. After the change above, there are no nested container-fluid elements any more, right? Just the two from app.jsx:
```
src/app.jsx:                <div key={"containerslists"} className="container-fluid">
src/app.jsx:                <div key={"imageslists"} className="container-fluid">
```

Reply:
This means style immediate sibling elements.

So if there are two `container-fluid`s in a row, remove the padding at the top of the second one, making the spacing between them the same as before and after (instead of doubled-up with both the bottom of the first and the top of the second).